The GLM library is already packed for Fedora; let's use it

--- EmptyEpsilon-EE-2021.06.23/CMakeLists.txt	2021-06-23 14:12:02.000000000 +0200
+++ EmptyEpsilon-EE-2021.06.23/CMakeLists.txt_patched	2021-06-26 04:54:53.932415114 +0200
@@ -106,17 +106,7 @@ if(WITH_DISCORD)
 endif()
 
 # GLM
-set(GLM_BASE_PATH "${EXTERNALS_DIR}")
-
-if(NOT EXISTS "${GLM_BASE_PATH}/glm/CMakeLists.txt")
-  set(GLM_ZIP "${DOWNLOADS_DIR}/glm.zip")
-  file(DOWNLOAD "https://github.com/g-truc/glm/releases/download/0.9.9.8/glm-0.9.9.8.zip" "${GLM_ZIP}" TIMEOUT 60 TLS_VERIFY ON)
-
-  file(MAKE_DIRECTORY "${GLM_BASE_PATH}/glm")
-  execute_process(COMMAND ${CMAKE_COMMAND} -E tar -xf "${GLM_ZIP}" WORKING_DIRECTORY "${GLM_BASE_PATH}")
-endif()
-
-add_subdirectory("${GLM_BASE_PATH}/glm" "${PROJECT_BINARY_DIR}/glm" EXCLUDE_FROM_ALL)
+find_package( glm REQUIRED )
 
 set(SOURCES
     src/main.cpp
